Packs/DSPM/Integrations/DSPM/DSPM.py

```diff
@@ -0,0 +1,903 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+from CommonServerUserPython import *  # noqa: F401
+
+import urllib3
+from typing import Any
+import re
+
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+""" CONSTANTS """
+MAX_PAGE_SIZE: int = 50
+DATE_FORMAT = "%Y-%m-%dT%H:%M:%SZ"  # ISO8601 format with UTC, default in XSOAR
+GET_RISK_FINDINGS_ENDPOINT = "/v1/risk-findings"
+GET_ASSET_LISTS = "/v1/assets"
+GET_ASSET_DETAILS = "/v1/assets/id?id="
+GET_ASSET_FILES = "/v1/classification/asset-files/id"
+GET_DATA_TYPES_ENDPOINT: str = "/v1/classification/data-types"
+GET_DATA_TYPE_FINDINGS_ENDPOINT: str = "/v1/data-type-findings"
+GET_ALERTS_LIST: str = "/v1/alerts"
+GET_RISK_FINDING_BY_ID: str = "/v1/risk-findings/id/"
+INCIDENT_STATUS = {"OPEN": 1, "INVESTIGATING": 2, "HANDLED": 2, "CLOSED": 2}
+RISK_STATUS = {"Active": "OPEN", "Closed": "INVESTIGATING", "Pending": "INVESTIGATING"}
+
+SUPPORTED_CLOUD_PROVIDERS = ["AWS", "AZURE", "GCP", "SNOWFLAKE", "FILE_SHARE", "O365"]
+SUPPORTED_CATEGORIES = [
+    "SECURITY",
+    "COMPLIANCE",
+    "GOVERNANCE",
+    "SECURITY_AND_COMPLIANCE",
+    "SECURITY_AND_GOVERNANCE",
+    "COMPLIANCE_AND_GOVERNANCE",
+    "SECURITY_AND_COMPLIANCE_AND_GOVERNANCE",
+]
+SUPPORTED_STATUSES = [
```

Comment:
remove **SUPPORTED_STATUSES**, you declare it twice

Reply:
Sure, will remove it.